stock_removal_location_by_priority/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Stock Removal Location by Priority",
+    "summary": "Establish a removal priority on stock locations.",
+    "version": "12.0.1.0.0",
+    "author": "Eficent, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse Management",
```

Comment:
@mreficent Change this to 'Warehouse' to get the good category in OCA Shop

Reply:
good!